css/leaderboard-style.css

```diff
@@ -0,0 +1,138 @@
+body{
+    
+    background:linear-gradient(rgba(0,0,0,0.6), rgba(0,0,0,0.8)),url("../images/leaderboard.jpg");
```

Comment:
What happens when the number of entries are larger than the size of the image ?
I think something like a gradient which is always changing colour would look cooler.

Reply:
This background image also looks good but in that case, we need to make leaderboard thing scrollable instead of the whole page getting scrolled, something like fixed background.
Also, @aayushjoglekar try using opencode's gradient with a dark overlay, I think it will look much better than image.

Reply:
or just do white. No background. 😄

Reply:
@gauthamzz 

People will count their total points themselves and update it in the "Total" key of their object in scores.json  
The leaderboard would look exactly the same as in the screenshot when more scores are updated.

Reply:
@gauthamzz 

Rank is nothing but the index number of each row after sorting objects in descending order of scores.